nix/cells/_writers/library.nix

```diff
@@ -0,0 +1,121 @@
+# stolen frome https://github.com/input-output-hk/bitte-cells/blob/main/cells/_writers/library.nix
```

Comment:
Thanks,  I see. I already opened a discussion to talk about a command library for that(under MIT) or write a library by myself. Until this time the std is still in the exploration stage even this PR.  moreover, I can sure it will be removed while my PR ready for PR.